### ISSUE TITLE
Noob help request

### ISSUE THREAD

**Issue by hganavakcinimod** [alicevision/Meshroom#1598]
Hi all, total noob here. 
I work in 3d dentistry applications (just recently started) and am trying to get a better understanding of how all this 3d cad and cam actually works. This software seems like a really good starting point as it *appears* to work in a similar way to the intraoral scanners for dentistry work - however, i've been running into some issues which i am sure are user error. I've been working on this project for a few days and have had a few false starts getting to this point, but now I can't go any further and the video tutorials don't seem to come up with anything that is helping me proceed. 
Does anyone know what this error message (screenshot below hopefully) means and what is going wrong? 
Many thanks!

![image](https://user-images.githubusercontent.com/97716908/149446832-3109e824-166c-42d0-837f-fd4222f7147b.png)

**Comment by fabiencastan** [comment]
Yes, looks like the output of the StructureFromMotion node is missing.
You should check the status of the StructureFromMotion node and the output log.

**Comment by Begalov** [comment]
meshroom is too sensitive for path symbols
try to avoid spaces in parent folder names
please report back if it helped!

**Comment by hganavakcinimod** [comment]
Hello! Thank you for your help. I was able I get the file to successfully generate by changing the source of the images from a cloud based drive to an sd card. I am not sure if this was the cause of the initial failure but it was successful. I also changed the camera type (initially images where jpeg on an iPhone to jpeg from a Nikon DSLR).
I was able to clean up the image in meshroom enough to export the stl to my printing software (I use a variety, primarily 3dSprint) but am now struggling to make it watertight, this is an issue for meshroom though I suspect!
Thank you for your help

**Comment by natowi** [comment]
`now struggling to make it watertight`

Meshlab could help

**Comment by cbentejac** [comment]
Closing this issue as it hasn't been updated in the past years.
